CHANGELOG.md

```diff
@@ -122,3 +122,6 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
 ## [2018.2] - 2018-02-16
 
 ## [2018.1] - 2018-02-05
+
+### Added
+- An uptane configuration parameter (force_install_completion) that triggers a system reboot if there is any pending updates that can be applied only after a reboot (the ostree package manager). Emulates a reboot in case of the fake package manager
```

Comment:
You've added this part in the wrong section. It should be at the top of the file in the `[??? (unreleased)]` section.

Also, we keep these fairly high level, so I would drop the part which mentions the fake package manager.

Would tweak it like that:

> A new uptane configuration parameter `force_install_completion` that triggers a system reboot at the end of the installation process for update mechanisms that need one to complete (e.g. OSTree package manager)

Reply:
OK. Never used it before.Read this https://keepachangelog.com/en/1.0.0/ and try follow it further.